.docker/intelmq-full-dev/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM certat/intelmq-full:1.0
+
+MAINTAINER Einar <elanfranco@cert.unlp.edu.ar>
+MAINTAINER Jeremias <jpretto@cert.unlp.edu.ar>
+
+USER root
+
+ADD entrypoint_dev.sh /opt/dev/entrypoint_dev.sh
+ADD update.sh /opt/dev/update.sh
+ADD merge_BOTS.py /opt/dev/merge_BOTS.py
```

Comment:
Ok, I understand.
But i just got tired of having json issues, and for prevent this we developed automixing. Now i changed the solution to be optional, and if the user dont want to enable automix, the script just copy the mounted to the container. 
This copy is necessary, because to install new bots, setup rely in https://github.com/CERTUNLP/intelmq-docker-1/blob/main/docker-compose-dev.yml#L27.

Reply:
Now you can use a variable in docker-compose-dev.yml